app/helpers/admin/unavailable_field_schedules_helper.rb

```diff
@@ -62,24 +62,26 @@ def format_schedule_date schedule
   def resolve_schedule_status_to_html schedule
     case schedule.status
     when "maintain"
```

Comment:
```suggestion
    case schedule.status.to_sym
    when :maintain
```
nên dùng symbol thay cho string ạ